packages/plugin-data/src/commands/force/data/tree/export.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+
+import { SfdxCommand } from '@salesforce/command';
+import { flags, FlagsConfig } from '@salesforce/command';
+import { Messages, Org } from '@salesforce/core';
+import { ExportApi } from '../../../../api/data/tree/exportApi';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-data', 'tree.export');
+
+export default class Export extends SfdxCommand {
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+  public static readonly requiresProject = false;
+  public static readonly requiresUsername = true;
+  public static readonly flagsConfig: FlagsConfig = {
+    query: flags.string({
+      char: 'q',
+      description: messages.getMessage('query'),
+      required: true,
+    }),
+    plan: flags.boolean({
+      char: 'p',
+      description: messages.getMessage('plan'),
+    }),
+    prefix: flags.string({
+      char: 'x',
+      description: messages.getMessage('prefix'),
+    }),
+    outputdir: flags.directory({
+      char: 'd',
+      description: messages.getMessage('outputdir'),
+    }),
+  };
+
+  // Overrides SfdxCommand.  This is ensured since requiresUsername == true
+  protected org!: Org;
```

Comment:
do we need to define this again? I thought it was brought in with SfdxCommad

Reply:
Unfortunately yes I think.  SfdxCommand.org is optional and to satisfy the TS compiler I had to override and tell it that `this.org` was not possibly undefined.  Let me know if you have another trick to handle this.

Reply:
This is the trick I have been using too.